test/truffle/docker/oraclelinux/Dockerfile

```diff
@@ -0,0 +1,42 @@
+FROM oraclelinux:7.3
+
+MAINTAINER chris.seaton@oracle.com
+
+# We need a conventional locale for testing
+ENV LANG=en_US.UTF-8
+
+# Tools we will need to get and run our tests
+RUN yum install -y git-1.8.3.1 which-2.20
+
+# Create a user and working directory
+WORKDIR /test
+RUN useradd -ms /bin/bash test
+RUN chown test /test
+USER test
+
+# Extract the GraalVM binary tarball
+ENV GRAALVM_VERSION=0.24
+ADD graalvm-$GRAALVM_VERSION-linux-amd64-jdk8.tar.gz /test
+ENV PATH=/test/graalvm-$GRAALVM_VERSION/bin:$PATH
+
+# Straight away, just run the ruby executable as the most basic test
+RUN ruby --version
+
+# Clone the TruffleRuby source code to get tests
+RUN git clone --depth 1 https://github.com/graalvm/truffleruby.git
```

Comment:
Yes, and in fact that's why I'm running such a small set of tests because the others don't work on the release. I need to get a tag for 2.4 first.